docs/source/implementations/feature_importance.rst

```diff
@@ -52,16 +52,22 @@ An example showing how to use various feature importance functions::
 
   oos_score = ml_cross_val_score(clf, X_train, y_train, cv_gen=cv_gen, sample_weight_train=None, scoring=accuracy_score).mean()
 
+  feature_clusters = get_feature_clusters(X_train, dependence_metric='information_variation', distance_metric='angular',
+                                          linkage_method='single', n_clusters=None)
+
   mdi_feature_imp = mean_decrease_impurity(clf, X_train.columns)
   mda_feature_imp = mean_decrease_accuracy(clf, X_train, y_train, cv_gen, scoring=log_loss)
   sfi_feature_imp = single_feature_importance(clf, X_train, y_train, cv_gen, scoring=accuracy_score)
+  cfi_feature_imp = mean_decrease_accuracy(clf, X_train, y_train, cv_gen, clustered_subsets=feature_clusters, scoring=log_loss)
 
   plot_feature_importance(mdi_feat_imp, oob_score=clf.oob_score_, oos_score=oos_score,
                                 save_fig=True, output_path='mdi_feat_imp.png')
   plot_feature_importance(mda_feat_imp, oob_score=clf.oob_score_, oos_score=oos_score,
                                 save_fig=True, output_path='mda_feat_imp.png')
   plot_feature_importance(sfi_feat_imp, oob_score=clf.oob_score_, oos_score=oos_score,
                                 save_fig=True, output_path='sfi_feat_imp.png')
+  plot_feature_importance(cfi_feat_imp, oob_score=clf.oob_score_, oos_score=oos_score,
```

Comment:
This whole file is outdated. Please pull from dev and handle merge conflicts.

Reply:
I have done the commit for the docs 4 days ago and there was changes made to the docs 2 days ago. So obviously that will make it outdated.

Reply:
Ah that makes sense. Thank you. I thought this was open this morning. I missed that.